201-cosmosdb-advanced-threat-protection-create-account/azuredeploy.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "defaultValue": "[uniqueString('cosmosdb', resourceGroup().id)]",
+      "metadata": {
+        "description": "The name of your CosmosDB account"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for your CosmosDB account"
+      }
+    },
+    "tier": {
+      "type": "string",
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "CosmosDB account tier"
+      }
+    },
+    "advancedThreatProtectionEnabled": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Enable or disable Advanced Threat Protection."
+      }
+    }
+  },
+  "resources": [
+    {
+      "name": "[parameters('name')]",
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "apiVersion": "2016-03-31",
+      "location": "[parameters('location')]",
+      "properties": {
+        "locations": [
+          {
+            "locationName": "[parameters('location')]"
+          }
+        ],
+        "databaseAccountOfferType": "[parameters('tier')]"
```

Comment:
this field called 'tier' at the top of the file, why there is a difference?

Reply:
mostly to be consistent with the basic cosmos template. at the top is a parameter from user (we can name it as we want) and this one is the field name in the json object to be deployed.